devel/libgit2/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           legacysupport 1.0
 PortGroup           muniversal 1.0
 
 # don't forget to update py-pygit2 and libgit2-glib as well
-github.setup        libgit2 libgit2 1.0.1 v
+github.setup        libgit2 libgit2 1.1.0 v
 revision            1
```

Comment:
When the version is increased, the revision should go back to 0.

Reply:
Didn't know that, thanks!